config_manager.go

```diff
@@ -35,6 +37,16 @@ type ConfigFile struct {
 	ExcludePatterns *RegexCollection `yaml:"exclude_patterns"`
 }
 
+func (cf ConfigFile) printValidationWarnings() {
+	if len(cf.Files) == 0 {
+		log.Warningf("Validation: configuration file contains no files to watch")
+	}
+	// Warn, that port is not set, only if host is specified
+	if cf.Destination.Host != "" && cf.Destination.Port == 0 {
+		log.Warningf("Validation: destination port is not specified in configuration file")
```

Comment:
@troy as we're not defaulting the destination host to `logs.papertrailapp.com`, should we also remove the default port and fail hard if it's not specified?